Tasks/QuickPerfTest/Invoke-QuickPerfTest.ps1

```diff
@@ -240,12 +260,18 @@ if ($drop.dropType -eq "InPlaceDrop")
     MonitorTestRun $h $run
 
     Write-Output ("Run-id for this load test is {0} and its name is '{1}'." -f  $run.runNumber, $run.name)
+    Write-Output ("To view run details navigate to {0}/_apps/hub/ms.vss-cloudloadtest-web.hub-loadtest-account?_a=summary&runId={1}" -f $tfsUrl, $run.id)
     Write-Output "To view detailed results navigate to Load Test | Load Test Manager in Visual Studio IDE, and open this run."
+
+    ("Run-id for this load test is {0} and its name is '{1}'." -f  $run.runNumber, $run.name) >>  $summaryFile
+    ("To view run details navigate [here]({0}/_apps/hub/ms.vss-cloudloadtest-web.hub-loadtest-account?_a=summary&runId={1})." -f $tfsUrl, $run.id) >>  $summaryFile
```

Comment:
since it is a md file, I guess we can improve the formatting. I'll sync up with you.